pkg/provider/provider.go

```diff
@@ -499,6 +500,30 @@ func (p *Pod) String() string {
 	)
 }
 
+func (p *Pod) IsCPUIsolationEnabled() bool {
+	return LoadBalancingDisabled(p) && AreResourcesIdentical(p)
```

Comment:
I would change it to the following:
return AreCPUResourcesWholeUnits(p) && AreResourcesIdentical(p)
name of the function IsCPUIsolationEnabled -> IsPodGuaranteed

Reply:
Adjusted.